buddyboss/buddyboss-platform
# 'Mismatch' notice on Email profile field on Register page.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #496.

### How to test the changes in this Pull Request:

1. Place 'Profile field' field on register page as optional.
2. Place a required field on register page.
3. Fill the register page.
4. Do not fill the required field.
5. See the notice on email field.

### Proof Screenshots or Video
https://screencast-o-matic.com/watch/cYnVj4wQJ7


### Other information:
